pcdsdevices/signal.py

```diff
@@ -293,3 +297,63 @@ def put(self, value, *, timestamp=None, force=False):
 
     def set(self, value, *, timestamp=None, force=False):
         return Signal.set(self, value, timestamp=timestamp, force=force)
+
+
+class UnitConversionDerivedSignal(DerivedSignal):
+    """
+    A DerivedSignal which performs unit conversion.
+
+    Custom units may be specified for the original signal, or if specified, the
+    original signal's units may be retrieved upon first connection.
+
+
+    Parameters
+    ----------
+    derived_from : Signal or str
+        The signal from which this one is derived.  This may be a string
+        attribute name that indicates a sibling to use.  When used in a
+        ``Device``, this is then simply the attribute name of another
+        ``Component``.
+
+    write_access : bool, optional
+        Write access may be disabled by setting this to ``False``, regardless
+        of the write access of the underlying signal.
+
+    name : str, optional
+        The signal name.
+
+    parent : Device, optional
+        The parent device.  Required if ``derived_from`` is an attribute name.
+    """
```

Comment:
Looks like this documents the *args and **kwargs, should also add in the `derived_units` and `original_units` too

Reply:
D'oh, thanks - good catch. Lazy docstring writing.